Tasks/Common/TlsHelper_/TlsHelper_.psm1

```diff
@@ -0,0 +1,12 @@
+function Add-Tls12InSession {
+    [CmdletBinding()]
+    param()
+
+    try {
+        [Net.ServicePointManager]::SecurityProtocol += [Net.SecurityProtocolType]::Tls12
```

Comment:
Somewhere I read that this enum is not exposed for lower .net version and hence an alternative is to use integer value directly

Reply:
Fixed